### PR TITLE
Added Remote BMC accounts and roles

### PR DESCRIPTION
**Pull request by sunithasomanna**
- Added a new section "Remote BMC accounts and roles" in the API readme.
- Changed the heading level for 2 VirtualMedia topics.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -7159,6 +7168,430 @@ curl -i POST \
...
+'https://{odim_host}:{port}/redfish/v1/Managers/{ManagerId}/RemoteAccountService/Accounts/{AccountID}
+```
+
+
+> **Sample request body**
+
+```
+{
+"RoleId":"Operator",
+"Password": "HP1nvent1"
```
Comment (amar-shalgar) [inline docs/README.md]: Please add a standard password across requests

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md]: Addressed

File: docs/README.md [hunk 2]
```diff
@@ -7159,6 +7168,430 @@ curl -i POST \
...
+
+| API URI                                                      | Operation Applicable | Required privileges            |
+| ------------------------------------------------------------ | -------------------- | ------------------------------ |
+| /redfish/v1/Managers/{ManagerId}/RemoteAccountService        | GET                  | `Login`                        |
+| /redfish/v1/Managers/{ManagerId}/RemoteAccountService/Accounts | GET, POST            | `Login`, `ConfigureComponents` |
+| /redfish/v1/Managers/{ManagerId}/RemoteAccountService/Accounts/{AccountId} | GET, PATCH, DELETE   | `Login`, `ConfigureComponents` |
+| /redfish/v1/Managers/{ManagerId}/RemoteAccountService/Roles  | GET                  | `Login`                        |
+| /redfish/v1/Managers/{ManagerId}/RemoteAccountService/Roles/{roleid} | GET                  | `Login`                        |
+
+<blockquote> NOTE: Before accessing these endpoints, ensure that the user has the required privileges. If you access these endpoints without necessary privileges, you will receive an HTTP `403 Forbidden` error. </blockquote>
```
Comment (amar-shalgar) [inline docs/README.md:7189]: This sentence should be - "you will receive a HTTP" instead of "you will receive an HTTP"

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

**Review by amar-shalgar — APPROVED** [review]

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md:7189]: Explained rules of articles usage

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**